### PR TITLE
Fix 1535: Don't set camera position for Null vector

### PR DESCRIPTION
**Pull request by darkwing**
Took a stab at #1535 ; reuses existing function, avoids unnecessary processing.

### CODE REVIEW

File: src/utils/vectors.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+module.exports.isNullVector = function (vector) {
```
Comment (ngokevin) [inline src/utils/vectors.js]: You could put this in `utils/coordinates.js` where all the other vector stuff is.

**Comment by ngokevin** [comment]
brave enough for a test? i spent a lot of today testing look-controls, would be great to have a start on wasd-controls. i'd mimic https://github.com/aframevr/aframe/pull/1853/files#diff-387522ff670c30773ff46795b9f7dc84R7 where tests are in `core/controls.test.js` and we integration test using the default camera.

**Comment by ngokevin** [comment]
could use a small test

**Comment by darkwing** [comment]
I plan to do so early this week.

**Comment by darkwing** [comment]
Starting these tests tonight.

**Comment by darkwing** [comment]
I was writing a test for this today, it kept failing, so upon further review I don't think this is needed due to @ngokevin's recent PR and this line: https://github.com/aframevr/aframe/pull/1904/files#diff-6231789e7e7fac0fbea47834d09b3bdcR56

Let me know if I'm wrong but I think this can be closed.

**Comment by ngokevin** [comment]
Cool. Thanks. I was just writing the tests to accompany the look-controls and guess I accidentally did this too.

**Pull request closed without merging**